src/engine/guess_toolset.bat

```diff
@@ -1,4 +1,5 @@
-@ECHO OFF
+@IF NOT DEFINED DEBUG_BATCH @ECHO OFF
```

Comment:
BTW: stole the concept from VS2017 `VsDevCmd.bat` line 3: `@if NOT "%VSCMD_DEBUG%" GEQ "3" @echo off`